src/shed/_codemods.py

```diff
@@ -332,3 +332,49 @@ def _has_none(node):
             return updated_node.with_changes(left=updated_node.right, right=node_left)
         else:
             return updated_node
+
+    @m.call_if_inside(m.Annotation(annotation=m.BinaryOperation()))
+    @m.leave(
+        m.BinaryOperation(
+            left=m.Subscript(value=m.Name(value="Literal")),
+            operator=m.BitOr(),
+            right=m.Name("None"),
+        )
+    )
+    def flatten_literal_op(self, _, updated_node):
+        literal = updated_node.left
+        args = list(literal.slice)
+        for item in args:
+            if m.matches(item, m.SubscriptElement(m.Index(m.Name("None")))):
+                return literal  # Already has "None"
+        args[-1] = args[-1].with_changes(comma=cst.Comma())
```

Comment:
It looks like we don't actually need this line, because [the last argument must have either a comma, or a `MaybeSentinel`](https://libcst.readthedocs.io/en/latest/nodes.html#libcst.SubscriptElement) which will be replaced with a comma on serialization.